src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -280,8 +280,8 @@
     <Compile Include="$(BclSourcesRoot)\System\Runtime\InteropServices\CustomMarshalers\EnumerableViewOfDispatch.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Runtime\InteropServices\CustomMarshalers\EnumeratorToEnumVariantMarshaler.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Runtime\InteropServices\CustomMarshalers\EnumeratorViewOfEnumVariant.cs" />
-    <Compile Include="$(BclSourcesRoot)\System\Runtime\InteropServices\CustomMarshalers\ExpandoToDispatchExMarshaler.cs" />
-    <Compile Include="$(BclSourcesRoot)\System\Runtime\InteropServices\CustomMarshalers\TypeToTypeInfoMarshaler.cs" />
+    <Compile Include="$(CommonPath)\System\Runtime\InteropServices\CustomMarshalers\ExpandoToDispatchExMarshaler.cs" />
```

Comment:
I do not think we want to add these to NativeAOT just to make some random tests pass. 

These are part of built-in COM interop support. We do not have built-in support in NativeAOT. It does not make sense to add a small incoherent bits of it.

Reply:
So I should undo changes and mark test as CoreCLR specific?

Reply:
It would be best to mark it as built-in COM interop specific using `PlatformDetection.SupportsComInterop`.

Reply:
Thank you.